h2o-r/tests/testdir_jira/runit_pubdev_3847.R

```diff
@@ -0,0 +1,30 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+######################################################################################
+# Turn the PUBDEV-3847 issue into the test and check if it fails
+######################################################################################
+pubdev.3847.test <-
+  function() {
+      file <- locate("smalldata/jira/pubdev_3847.csv")
+      data <- h2o.importFile(file, destination_frame = "pubdev3847.data", parse = FALSE)
```

Comment:
This is causing the test failure. Please remove `parse = FALSE` - it should resolve it.